src/utils/path.ts

```diff
@@ -0,0 +1,15 @@
+import { homedir } from 'os';
+import { resolve } from 'path';
+
+export function getConfigPath() {
```

Comment:
If module has single function named some way, we should name the module with the name of the function.

Reply:
I generally never do that so it's easier to know what a module is about by looking at the file structure (`utilities/path` -> it's utilities about paths), and more methods may get added into the module later on. I would agree if it was the default export, but utilities tend to have a general scope IMO

Reply:
I don't find this good use case for utility module. Utility methods should be universal. This is just a method returning the path of the config. So it can be simply moved to file called "configPath". Utility file like this will tend to grow. And sooner or later it will have 1000 lines of methods doing "something" with path. What we want is small files with few lines of code, which are logically distributed in directories.